source/dub/dependency.d

```diff
@@ -33,9 +33,9 @@ struct Dependency {
 	private {
 		// Shortcut to create >=0.0.0
 		enum ANY_IDENT = "*";
-		string m_cmpA;
+		bool m_inclusiveA = true; // A comparison > (true) or >= (false)
 		Version m_versA;
-		string m_cmpB;
+		bool m_inclusiveB = true; // B comparison < (true) or <= (false)
```

Comment:
I'd use mathematical boundaries to describe this, as in `"[)"`.
We use that in [uniform](http://dlang.org/phobos/std_random.html#.uniform) and independently from that I came up with it as well in my graphics library.
https://github.com/MartinNowak/graphics/blob/ea9c7fcea56993d8f48aef3b0f602f5a92c783bf/src/graphics/math/clamp.d#L19
You also only need `<`, `<=` and `==` by swapping the arguments to `>` or `>=`, as in the linked code above.